locales/base/translation.json

```diff
@@ -107,7 +107,7 @@
   },
   "signInWithEmail": {
     "title": "Sign in with email",
-    "subtitle": "Sign in with your email to set it up as a backup. You will be prompted to sign in again if you ever lose your wallet.",
+    "subtitle": "Remember this email address & password. You will be prompted to sign in again if you ever lose your wallet.",
```

Comment:
Looks like we have different copies for CAB setup in onboarding vs settings per figma, I wonder if this was a bug in Valora (i.e., when we added CAB setup in onboarding we never did the different copies for this screen). Maybe we should fix this in Valora?

Reply:
Hmm, I asked Kayla about this, will leave this as-is till then.